api/types/container/config.go

```diff
@@ -19,8 +19,9 @@ type HealthConfig struct {
 	Test []string `json:",omitempty"`
 
 	// Zero means to inherit. Durations are expressed as integer nanoseconds.
-	Interval time.Duration `json:",omitempty"` // Interval is the time to wait between checks.
-	Timeout  time.Duration `json:",omitempty"` // Timeout is the time to wait before considering the check to have hung.
+	Interval    time.Duration `json:",omitempty"` // Interval is the time to wait between checks.
+	Timeout     time.Duration `json:",omitempty"` // Timeout is the time to wait before considering the check to have hung.
+	StartPeriod time.Duration `json:",omitempty"` // The start period for the container to initialize before the retries starts to count down.
```

Comment:
Can you update the swagger.yml to include the new field? https://github.com/docker/docker/blob/9f9cc221bdb0a919d6ed28bdfaffa9ae2fe3a5fd/api/swagger.yaml#L485-L508

Also can you add a line to the API version-history; https://github.com/docker/docker/blob/master/docs/api/version-history.md#v129-api-changes